README.md

```diff
@@ -123,28 +114,62 @@ This requires setting up [binfmt_misc](https://docs.kernel.org/admin-guide/binfm
 On most distributions, you can install QEMU user static to set up binfmt_misc handlers. For example, on Debian, use the command `apt install qemu-user-static`.
 
 
-## Customizing
-Building Garden Linux is based on a [feature system](features/README.md).
+### SELinux
 
-| Feature Type | Includes |
-|---|---|
-| Platforms | `ali`, `aws`, `azure`, `gcp`, `kvm`, `metal`, ... |
-| Features | `container host`, `virtual host`, ... |
-| Modifiers | `_slim`, `_readonly`, `_pxe`, `_iso`, ... |
-| Element | `cis`, `fedramp`, `gardener` |
+If you intend to build targets with the `_selinux` feature some additional requirements apply to the build machine.
+Building the `_selinux` feature will not work on machines running in SELinux enforcing mode. Ideally you should build on a build machine with SELinux disabled, but if you want to build with SELinux in permissive mode this can be achieved by running build as root with the `--privileged` flag.
 
-if you want to build manually choose:
-```
-build.sh  --features <Platform>,[<feature1>],[<featureX>],[_modifier1],[_modifierX] destination [version]
-```
+i.e.: `sudo ./build --privileged ${target}`
 
-Additionally, please find some `build.sh` example calls in the `Makefile`.
+### Secureboot
+
+If you intend to build targets with the `_secureboot` feature, you must first build the secureboot certificates.
+Run the command `./cert/build` to generate the secureboot certificates.
+
+By default, the command uses local files as the private key storage. However, you can configure it to use the AWS KMS key store by using the `--kms` flag. Note that valid AWS credentials need to be configured using the standard AWS environment variables.
+
+### Maintained Images
+
+Garden Linux offers a range of official images tailored for various architectures and platforms. Instead of listing all the specific image variants here, you can find the most up-to-date list of maintained images on the [Garden Linux releases page on GitHub](https://github.com/gardenlinux/gardenlinux/releases).
+
+**Supported Architectures:**
+- arm64
+- amd64
+
+**Supported Platforms:**
+- Ali
+- AWS
+- Azure
+- GCP
+- KVM
+- Metal
+- OpenStack
+- VMware
```

Comment:
question: is spelling here case sensitive in the `./build $plattform` cli? Maybe it would be good here to add the technical identifier like

```
- Ali (`ali`)
- AWS (`aws`)
...
```

or something like that